Static list component (#3975)
jpmorganchase/salt-ds: site/docs/components/static-list/usage.mdx

---
title:
  $ref: ./#/title
layout: DetailComponent
sidebar:
  exclude: true
data:
  $ref: ./#/data
---

## Using the component

### When to use

> - To present a set of list items which are static. It can be used in conjunction with File drop zone to create a file uploader pattern to show the list of files added, it can also be used to create a pattern for cart details.

Reviewer: This description goes straight to File Dropdown and File Uploader pattern, which we do not have.
Can we remove that and as per description, reference them as example use-cases but not so specific.

Ah ... we use cart here but in my comment, I said 'checkout basket'.. Need to align that
So, we either use 'shopping cart' or 'checkout basket'.
I don't have a preference, as long as the language aligns.

It should be bias to US locale, does 'checkout basket' work in the US, or is 'shopping cart' more familiar?

> - To present a set of list items which are static. It can be used in conjunction with File drop zone to create a file uploader pattern to show the list of files added, it can also be used to create a pattern for cart details.

Reviewer 2: ```suggestion
- To present a set of static list items. It can be used with a File drop zone to show the list of files added. It can also be used to create a shopping cart list.
```

### When not to use

> - If you need to create a interactive list, use the [`ListBox`](../list-box) component.

Reviewer 2: ```suggestion
- If you need to create an interactive list, use the [`ListBox`](../list-box) component.
```

## Content

- Always use sentence case for static list item labels.
- Keep static list item labels short and concise.
- Keep static list secondary item labels short and concise.

## Import

To import `StaticList` and related components from the lab Salt package, use:

```
import { StaticList, StaticListItem } from "@salt-ds/lab";
```

> ## Props

Reviewer 2: missing the content component

Author: added

Reviewer 3: removed it again with the header because tables are empty
